docs/guides/modules/orchestrate/pages/set-up-triggers.adoc

```diff
@@ -0,0 +1,121 @@
+= Set up triggers
+:page-platform: Cloud, Server v4+
+:page-description: A guide to setting up triggers for your pipelines.
+:experimental:
+
+Triggers are configurable rules that determine when to automatically run a pipeline based on specific events and conditions.
+
+A project can have one or more triggers, which are associated with events from a source of change. The most common type of trigger is a VCS event, for example, a push to a branch on a GitHub repository. Any system that can emit outbound webhooks can be set up as a trigger. A trigger can have only one pipeline but a pipeline can have multiple triggers.
```

Comment:
We should frame triggers as attributes of a **pipeline**, not a project. A trigger cannot exist if not associated to a pipeline. 
```suggestion
A pipeline can have one or more triggers, which are associated with events from a source of change. The most common type of trigger is a VCS event, for example, a push to a branch on a GitHub repository. Any system that can emit outbound webhooks can be set up as a trigger. A trigger can have only one pipeline but a pipeline can have multiple triggers.
```